daml-lf/verification/stainless.nix

```diff
@@ -0,0 +1,17 @@
+let
```

Comment:
I've added this file to simplify developer setup and so try and ensure some repeatability with this work

Reply:
Nice!
I don't know much about nix but would it be hard to also make it so that stainless-dotty is in the path as well?

Reply:
Ideally, we'd just reference a standard stainless/dotty release. However, due to a bug in stainless(?), the current work requires that we manually build a current HEAD version of stainless.

I understand that Andrea is going to prod the stainless team to see if he can get a newer release generated?

Reply:
So its likely not worth the effort to do something like this as we should be able to hook into more standard releases in the future.